ui/job-view/headerbars/HealthMenu.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Label } from 'reactstrap';
+
+export const PUSH_HEALTH_VISIBILITY = 'pushHealthVisibility';
+
+class HealthMenu extends PureComponent {
+  render() {
+    const { pushHealthVisibility, setPushHealthVisibility } = this.props;
+
+    return (
+      <span className="dropdown">
+        <span
+          id="healthLabel"
+          role="button"
+          title="Change visibility of the Push Health badge/link"
+          data-toggle="dropdown"
+          className="btn btn-view-nav btn-sm nav-menu-btn dropdown-toggle"
+        >
+          Health
+        </span>
+        <ul className="dropdown-menu checkbox-dropdown-menu" role="menu">
```

Comment:
You could also use the reactstrap dropdown menu: https://reactstrap.github.io/components/dropdowns/

Reply:
I realize you already have checkboxes being used for existing dropdown menu items, but if you wanted to try something else in the future the `DropdownMenuItems` component uses a checkbox icon (in shared folder) :)

Reply:
Boy, I need to check in that shared component folder more often.  These are both great!  Thanks for the tips.  :)

Reply:
This list doesn't match the other menus as well anymore, but it's worth converting the others to use this component.  It's nice and clean to use.